### PR TITLE
Various stuff

### LINKED ISSUE #1917: Particle#getColorMultiplier should be Particle#getBrightness
`20w49a+build.11`

This method is used to determine the brightness of the particle, deferring to the lightmap coordinates at the particle's block position by default. The method argument should also be `tickDelta` not `tint`.

**`Particle`**
```java
protected int getColorMultiplier(float tint) {
	BlockPos blockPos = new BlockPos(this.x, this.y, this.z);
	if (this.world.isChunkLoaded(blockPos)) {
		return WorldRenderer.getLightmapCoordinates(this.world, blockPos);
	}
	return 0;
}
```
**`BillboardParticle#buildGeometry`**
```java
int p = this.getColorMultiplier(tickDelta);
```

### LINKED ISSUE #1986: ScreenHandler's quickCraftButton and quickCraftStage fields are swapped
As per title, the field names `ScreenHandler.quickCraftButton` and `ScreenHandler.quickCraftStage` are swapped.

Proof: `ScreenHandler.method_30010` sets `quickCraftButton` to the return value of `unpackQuickCraftButton`, and then uses it to determine the quick craft stage it's currently on.

This possibly happened as a result of 1a6f261a2ebd5ab5c5572489ceea54ecc5a2ae74.

### PR DESCRIPTION
**Pull request by YanisBft**
Closes #1917 
Closes #1986

### CODE REVIEW

File: mappings/net/minecraft/fluid/FlowableFluid.mapping [hunk 1]
```diff
@@ -48,12 +48,14 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowableFluid
...
 		ARG 6 flowToBlockState
+		ARG 7 fluidState
+		ARG 8 fluid
 	METHOD method_15739 getLevelDecreasePerBlock (Lnet/minecraft/class_4538;)I
 		ARG 1 world
 	METHOD method_15740 (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)I
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_15741 (Lnet/minecraft/class_3610;)I
+	METHOD method_15741 getBlockStateLevel (Lnet/minecraft/class_3610;)I
```
Comment (liach) [inline mappings/net/minecraft/fluid/FlowableFluid.mapping:58]:
```suggestion
	METHOD method_15741 getLevelOf (Lnet/minecraft/class_3610;)I
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/fluid/FlowableFluid.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:58]: This method is called in the `toBlockState` method of WaterFluid and LavaFluid, to set the "level" block state

Comment (liach) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:58]: How is this a setter when its descriptor is `int method_15741(BlockState)`?

Comment (YanisBft) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:58]: I meant this method is used to set "level" block state _in_ `toBlockState`
image

Comment (liach) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:58]: Oh, 3610 is fluid state

File: mappings/net/minecraft/fluid/Fluid.mapping [hunk 2]
```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 	FIELD field_15905 stateManager Lnet/minecraft/class_2689;
 	METHOD method_15774 getBucketItem ()Lnet/minecraft/class_1792;
 	METHOD method_15775 appendProperties (Lnet/minecraft/class_2689$class_2690;)V
+		ARG 1 states
```
Comment (Juuxel) [inline mappings/net/minecraft/fluid/Fluid.mapping]:
```suggestion
		ARG 1 builder
```
The builder isn't really a list of states but a list of properties, and `Block.appendProperties` already uses `builder` as well.

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
@YanisBft would you mind fixing #1886 as well? It appears very closely related to #1986, and would fit into a subproject of fixing bad screenhandler hotkey names.

**Comment by YanisBft** [comment]
Actually it's a duplicate of #1986, as the fields are swapped, not the methods

**Review by Gegy — APPROVED** [review]

**Pull request merged**